app/controllers/comments_controller.rb

```diff
@@ -41,7 +41,7 @@ def set_organization
     @commentable = if authorized_user.can_admin_system?
       Organization.find(params[:organization_id])
     else
-      current_organization
+      Current.organization
     end
   end
```

Comment:
I'm thinking that we'll want to extract `set_organization` out into a concern as this behavior will be needed on every controller after login.